Sources/PasscodeKit/Views/KeypadView.swift

```diff
@@ -30,14 +35,42 @@ struct KeypadView: View {
                 NumberButton(value: .text("9"), text: $text)
             }
             HStack(spacing: configuration.hSpacing) {
-                NumberButton(value: .blank, text: $text)
-                    .hidden()
-                    .disabled(true)
+                if passcodeEnv.manager.passcode?.isBiometricsEnabled ?? false {
+                    Button {
+                        onBiometry()
+                    } label: {
+                        ZStack {
+                            Image(systemName: biometricsImage)
+                                .resizable()
+                                .foregroundColor(configuration.foregroundColor)
+                                .frame(maxWidth: 40, maxHeight: 40)
+                        }
+                        .foregroundStyle(.primary)
+                    }
+                    .buttonStyle(.plain)
+                    .frame(maxWidth: 100, maxHeight: 100)
```

Comment:
Do you think it looks better to have the biometrics button without the circular background like the other buttons? Or should this button also apply `.background(.ultraThinMaterial.blendMode(.multiply))` and `.clipShape(Circle())` like the `NumberButton`?

Either way is fine by me

Reply:
@divadretlaw Usually it's without circle, thanks